scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -158,6 +169,50 @@ def _create_test_project(self):
                    self.obs_test_project_name, '--accept-in-hours', 720,
                    '-m', 'Auto delete after 30 days.')
 
+    def get_obsinfo_filename(self, package, service_def=None):
+        if not service_def:
+            service_def_cmd = sh.osc(
+                '-A', 'https://api.suse.de', 'cat',
+                self.obs_linked_project,
+                package.name,
+                '_service')
+            service_def = str(service_def_cmd)
+        root = ET.fromstring(service_def)
+        nodes = root.findall(
+            './service[@name="obs_scm"]/param[@name="filename"]')
+        if len(nodes) != 1 or not nodes[0].text:
+            raise ValueError(
+                "There needs to be exactly one obs_scm service filename"
+                " in https://build.suse.de/package/view_file/%s/%s/_service"
+                % (self.obs_linked_project, package.name))
+        return nodes[0].text
+
+    def is_current(self, package):
+        if package.has_applied_changes():
+            return False
+
+        obsinfo_filename = self.get_obsinfo_filename(package)
```

Comment:
You have to manually add the '.obsinfo' to the name returned here in the osc command below, so if this method not misnamed? E.g. perhaps get_obsinfp_basename() would be better, since you usually append a .obsinfo or .obscpio to the returned name which using it...

Reply:
done